pre_process.py

```diff
@@ -224,7 +224,7 @@ def process(tensor_type, input_path, output_dir_path, tensor_size, schedule_dict
         if dtype == "int":
             for idx, val in np.ndenumerate(tensor):
                 if val != 0:
-                    tensor[idx] = np.random.randint(1, 100)
```

Comment:
Probs, np.random.randint(-100, 100)
and if tensor[idx] == 0, set to 1

Reply:
fixed